api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -457,21 +454,10 @@ private DbUser updateDuccAgreement(
     ducc.setUserGivenName(dbUser.getGivenName());
     ducc.setUserInitials(initials);
     ducc.setCompletionTime(timestamp);
+    ducc.setUserNameOutOfDate(false);
```

Comment:
A newly created or updated DUCC does not have an out of date name, by definition.